pgml-dashboard/templates/components/github_icon.html

```diff
@@ -1,8 +1,8 @@
 <a class="d-flex align-items-center nav-link p-0 border-bottom-0" href="https://github.com/postgresml/postgresml">
-<% if let Ok(stars) = crate::utils::config::github_stars() { %>
-    <span class="badge github-badge"><p>Stars | <%= stars %></p></span>
-<% } %>
-    <svg width="35" height="35" viewBox="0 0 40 39" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <% if show_stars {%>
```

Comment:
The stars should not be hardcoded, we fetch them from Github on every build.

Reply:
I am not sure how this got taken out.  I put it back in.  We still need to conditionally show starts since the webapp does not show the stars.